pages/docs.html

```diff
@@ -1322,6 +1322,52 @@ <h1 class="main__heading p--1" id="modal">
                     </div>
                 </div>
             </article>
+
+
+            <h1 class="main__heading p--1" id="topnav">
```

Comment:
Underscore character is not used in class naming convention. You can use '-' instead. Do look into this.

Reply:
BEM (block__element--modifier) is a standard class naming convention and I have used it throughout CL.